.teamcity/_self/projects/WebApp.kt

```diff
@@ -846,13 +846,13 @@ object PreReleaseE2ETests : BuildType({
 				set +o errexit
 
 				# Run suite.
-				xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --maxWorkers=%JEST_E2E_WORKERS% --group=calypso-release
+				xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --maxWorkers=%JEST_E2E_WORKERS% --group=calypso-release-test
 
 				# Restore exit on error.
 				set -o errexit
 
 				# Retry failed tests only.
-				RETRY_COUNT=1 xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --maxWorkers=%JEST_E2E_WORKERS% --group=calypso-release --onlyFailures
+				RETRY_COUNT=1 xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --maxWorkers=%JEST_E2E_WORKERS% --group=calypso-release-test --onlyFailures
```

Comment:
**Fix (blocking):** I'm guessing this was a debugging change that stuck around, but this changes the group name here so no tests would run and it would fail!

Reply:
Yep, this is testing stuff. Sorry you had to see it, I forgot that I had pushed it before marking it ready to review.